docs/trustee-tpm-setup.md

```diff
@@ -0,0 +1,203 @@
+```sh
+cat > kbs/test/dummy_data << EOF
+1234567890abcde
+EOF
+```
+
+Upload the resource to Trustee
+
+```sh
+kbs-client --url http://"$KBS_SERVER":8080 \
+           config --auth-private-key kbs/config/private.key \
+           set-resource --resource-file kbs/test/dummy_data \
+           --path default/test/dummy
+```
+
+### Set Attestation Policy
+
+This is an example attestation policy using `tpm_svn` and `tpm_pcr11` as part of TPM TCB.
+Adapt is as needed based on the claims you want to validate.
+
+Create the attestation policy file.
+
+```sh
+cat << 'EOF' > tpm_policy_cpu.rego
+package policy
+
+import rego.v1
+
+default hardware := 97
+default configuration := 36
+
+##### Sample
+
+
+executables := 3 if {
+	input.sample.launch_digest in data.reference.launch_digest
+}
+
+hardware := 2 if {
+	input.sample.svn in data.reference.svn
+	input.sample.platform_version.major == data.reference.major_version
+	input.sample.platform_version.minor >= data.reference.minimum_minor_version
+}
+
+##### TPM
+hardware := 2 if {
+	input.tpm.svn in data.reference.tpm_svn
+}
+
+executables := 3 if {
+	input.tpm.pcrs[11] in data.reference.tpm_pcr11
+}
+
+configuration := 2 if {
+	input.tpm.pcrs[11] in data.reference.tpm_pcr11
+}
+EOF
+
+```
+
+Set the attestation policy in Trustee.
+
+```sh
+kbs-client --url http://"$KBS_SERVER":8080 \
+           config --auth-private-key kbs/config/private.key \
+           set-attestation-policy \
+           --policy-file tpm_policy_cpu.rego
+```
+
+### Set Resource Policy
+
+Set an affirming resource policy.
+
+```sh
+kbs-client --url http://"$KBS_SERVER":8080 \
+           config --auth-private-key kbs/config/private.key \
+           set-resource-policy --affirming
+```
+
+### Register reference values for use with attestation
+
+Following example sets reference for `tpm_pcr11` and `tpm_svn`, which is used
+in the attestation policy
+
+```sh
+export PCR11=<set-digest-value>
+kbs-client --url http://"$KBS_SERVER":8080 \
+           config --auth-private-key kbs/config/private.key \
+           set-sample-reference-value tpm_pcr11 "$PCR11"
+
+kbs-client --url http://"$KBS_SERVER":8080 \
+           config --auth-private-key kbs/config/private.key \
+           set-sample-reference-value tpm_svn "1"
+```
+
+Verify the reference values
+
+```sh
+kbs-client --url http://"$KBS_SERVER":8080 \
+           config --auth-private-key kbs/config/private.key \
+           get-reference-values
+```
+
+### Verify
+
+Run the following to retrieve the resource.
+
+```sh
+sudo kbs-client --url http://"$KBS_SERVER":8080 \
+     get-resource --path default/test/dummy
+```
+
+If everything is setup correctly you should see the base64 encoded dummy data returned.
+
+```sh
+MTIzNDU2Nzg5MGFiY2RlCg==
+```
+
+## Trustee client setup
+
+You'll need to copy the `kbs-client` to the client system having TPM.
+
+Following command copies the kbs-client from Trustee host to a CoreOS VM.
+
+```sh
+export SSH_KEY_PATH=<set>
+export COREOS_VM_IP=<set>
```

Comment:
If you forward the ssh port, then it means that the VM is usually using user networking, so you can simply reference it as localhost

Reply:
Yeah. Since there could be different way to do the same thing, may be we can just keep the information generic. Wdyt?

Reply:
ok, fine for me